common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -213,14 +195,32 @@ trait CommercialSwitches {
   )
 
   val BookLookupSwitch = Switch(
-    SwitchGroup.Commercial,
+    SwitchGroup.CommercialFeeds,
     "book-lookup",
     "If this switch is on, book data will be looked up using a third-party service.",
     safeState = Off,
     sellByDate = never,
     exposeClientSide = false
   )
 
+  val MembersAreaSwitch = Switch(
```

Comment:
Actually ignore that - it is just because I moved some of the Commercial switches around, the diff is misleading; these switches are not new in my commit.